### DISCUSSION TITLE
How access quasar instance in boot files?

### DISCUSSION THREAD

**Discussion by redigaffi** [quasarframework/quasar#13327] [General - Components / Directives / etc] [answered]
How can I access the quasar instance in boot files?

**Comment by metalsadman** [comment, marked as answer]
use the passed param `app` in your boot function see https://quasar.dev/quasar-cli-vite/boot-files#anatomy-of-a-boot-file.

then inside you can access the global 
`app.config.globalProperties.$q`

**Reply by redigaffi** [reply]
that works, thanks
